pose-detection/src/blazepose_tfjs/detector.ts

```diff
@@ -299,44 +322,47 @@ class BlazePoseTfjsDetector implements PoseDetector {
 
     // PoseLandmarksByRoiCPU: InferenceCalculator
     // The model returns 5 tensor with the following shape:
-    // Full model:
-    // Output[4]: This tensor (shape: [1, 195]) represents 39 5-d keypoints.
-    // The first 33 refer to the keypoints. The final 6 key points refer to
-    // the alignment points from the detector model and the hands.)
-    // Output [3]: This tensor (shape: [1, 1]) represents the confidence
-    // score.
-    // Output [2]: This tensor (shape: [1, 64, 64, 39]) represents heatmap for
-    // the 39 landmarks.
     // Lite model:
-    // Output[4]: This tensor (shape: [1, 195]) represents 39 5-d keypoints.
-    // Output[0]: This tensor (shape: [1, 1]) represents the confidence score.
-    // Output[3]: This tensor (shape: [1, 64, 64, 39]) represents heatmap for
+    // Output[3]: This tensor (shape: [1, 195]) represents 39 5-d keypoints.
+    // Output[1]: This tensor (shape: [1, 1]) represents the confidence score.
+    // Output[2]: This tensor (shape: [1, 64, 64, 39]) represents heatmap for
     // the 39 landmarks.
+    // Output[0]: This tensor (shape: [1, 117]) represents 39 3DWorld keypoints.
+    // Full model:
+    // Output[3]: This tensor (shape: [1, 195]) represents 39 5-d keypoints.
+    // Output[1]: This tensor (shape: [1, 1]) represents the confidence score.
+    // Output[2]: This tensor (shape: [1, 64, 64, 39]) represents heatmap for
+    // the 39 landmarks.
+    // Output[0]: This tensor (shape: [1, 117]) represents 39 3DWorld keypoints.
     // Heavy model:
-    // Output[2]: This tensor (shape: [1, 195]) represents 39 5-d keypoints.
-    // Output[4]: This tensor (shape: [1, 1]) represents the confidence score.
-    // Output[1]: This tensor (shape: [1, 64, 64, 39]) represents heatmap for
+    // Output[0]: This tensor (shape: [1, 195]) represents 39 5-d keypoints.
```

Comment:
Why output indexes are different in lite/full/heavy?

Reply:
The old converter we used to turn the model into TFJS didn't keep the output name and it didn't retain the outputs in a fixed order each time so this had to be hardcoded in. The new converter fixes that with named outputs and the update is found in the Update model to named outputs commit